similarity_search/test/test_distfunc.cc

```diff
@@ -785,8 +785,8 @@ bool TestSparseCosineSimilarityAgree(const string& dataFile, size_t N, size_t Re
 
     bool bug = false;
 
-    float maxRelDiff = 1e-6f;
-    float maxAbsDiff = 1e-6f;
+    float maxRelDiff = 1e-5f;
+    float maxAbsDiff = 1e-5f;
```

Comment:
Yes, I can totally see the point, but I currently have no good idea how one can reliably test floating-point code like this one. In fact, floating point is often non-deterministic, i.e., results may differ from run to run on a single machine.